### PR TITLE
Reproducing #6212 

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
Ref https://github.com/sbt/sbt/issues/6212

Test

### CODE REVIEW

File: launcher-package/citest/test.bat [hunk 1]
```diff
@@ -8,6 +8,7 @@ unzip ..\target\universal\sbt.zip -d freshly-baked
 SETLOCAL
 
 "freshly-baked\sbt\bin\sbt" about
+"freshly-baked\sbt\bin\sbt" -Dsbt.repository.config=.\repositories -Dsbt.version=0.100 about
```
Comment (eed3si9n) [inline launcher-package/citest/test.bat]: I think `=` sign goes way without quotation:

```suggestion
"freshly-baked\sbt\bin\sbt" "-Dsbt.repository.config=.\repositories" "-Dsbt.version=0.100" about
```

Comment (Nirvikalpa108) [reply launcher-package/citest/test.bat]: Thanks 👍🏽 will try this now

**Comment by eed3si9n** [comment]
I'm confused why this doesn't show up in the log at all:

```
2021-08-28T04:56:35.8340293Z D:\a\sbt\sbt\launcher-package\citest>mkdir freshly-baked 
2021-08-28T04:56:35.8343843Z 
2021-08-28T04:56:35.8344880Z D:\a\sbt\sbt\launcher-package\citest>unzip ..\target\universal\sbt.zip -d freshly-baked 
2021-08-28T04:56:35.8980276Z Archive:  ..\target\universal\sbt.zip
2021-08-28T04:56:35.8993511Z   inflating: freshly-baked/sbt/bin/sbt.bat  
2021-08-28T04:56:35.9001064Z   inflating: freshly-baked/sbt/conf/sbtconfig.txt  
2021-08-28T04:56:35.9007783Z   inflating: freshly-baked/sbt/conf/sbtopts  
2021-08-28T04:56:35.9016699Z   inflating: freshly-baked/sbt/bin/sbt  
2021-08-28T04:56:35.9360483Z   inflating: freshly-baked/sbt/bin/sbt-launch.jar  
2021-08-28T04:56:36.0958657Z   inflating: freshly-baked/sbt/bin/sbtn-x86_64-pc-win32.exe  
2021-08-28T04:56:36.0964929Z   inflating: freshly-baked/sbt/LICENSE  
2021-08-28T04:56:36.0970668Z   inflating: freshly-baked/sbt/NOTICE  
2021-08-28T04:56:36.0989064Z 
2021-08-28T04:56:36.0989690Z D:\a\sbt\sbt\launcher-package\citest>SETLOCAL
2021-08-28T04:56:36.0992271Z 
2021-08-28T04:56:36.0993219Z D:\a\sbt\sbt\launcher-package\citest>"freshly-baked\sbt\bin\sbt" about 
2021-08-28T04:56:42.4281641Z [info] welcome to sbt 1.3.13 (AdoptOpenJDK Java 1.8.0_292)
2021-08-28T04:56:44.5698541Z [info] loading project definition from D:\a\sbt\sbt\launcher-package\citest\project
2021-08-28T04:56:46.2492259Z [info] loading settings for project root from build.sbt ...
2021-08-28T04:56:46.3981305Z [info] set current project to Hello (in build file:/D:/a/sbt/sbt/launcher-package/citest/)
2021-08-28T04:56:46.9793734Z [info] This is sbt 1.3.13
2021-08-28T04:56:46.9796085Z [info] The current project is ProjectRef(uri("file:/D:/a/sbt/sbt/launcher-package/citest/"), "root") 0.1.0-SNAPSHOT
2021-08-28T04:56:46.9797707Z [info] The current project is built against Scala 2.12.4
2021-08-28T04:56:46.9799746Z [info] Available Plugins
2021-08-28T04:56:46.9842098Z [info]  - sbt.ScriptedPlugin
2021-08-28T04:56:46.9842933Z [info]  - sbt.plugins.CorePlugin
2021-08-28T04:56:46.9843843Z [info]  - sbt.plugins.Giter8TemplatePlugin
2021-08-28T04:56:46.9844721Z [info]  - sbt.plugins.IvyPlugin
2021-08-28T04:56:46.9845684Z [info]  - sbt.plugins.JUnitXmlReportPlugin
2021-08-28T04:56:46.9846526Z [info]  - sbt.plugins.JvmPlugin
2021-08-28T04:56:46.9847241Z [info]  - sbt.plugins.SbtPlugin
2021-08-28T04:56:46.9848027Z [info]  - sbt.plugins.SemanticdbPlugin
2021-08-28T04:56:46.9848854Z [info] sbt, sbt plugins, and build definitions are using Scala 2.12.10
2021-08-28T04:56:47.1258340Z 
2021-08-28T04:56:47.1260100Z D:\a\sbt\sbt\launcher-package\citest>SETLOCAL
2021-08-28T04:56:47.1260987Z 
2021-08-28T04:56:47.1262195Z D:\a\sbt\sbt\launcher-package\citest>SET JAVA_HOME=C:\jdk11 
```

**Comment by eed3si9n** [comment]
Closing this for now.

**Pull request closed without merging**